roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -17,22 +17,15 @@ We account for these requirements by having the disputes module handle two kinds
 1. Local disputes: those contesting the validity of the current fork by disputing a parablock included within it.
 1. Remote disputes: a dispute that has partially or fully resolved on another fork which is transplanted to the local fork for completion and eventual slashing.
 
-When handling disputes on one chain,
+When a local dispute concludes negatively, the chain needs to be abandoned and reverted back to a block where the state does not contain the bad parablock. We expect that due to the [Approval Checking Protocol](../protocol-approval.md), the current executing block should not be finalized. So we do two things when a local dispute concludes negatively:
+1. Freeze the state of parachains so nothing further is backed or included.
+1. Issue a digest in the header of the block that signals to nodes that this branch of the chain is to be abandoned.
 
-## Storage
-
-Helpers:
+If, as is expected, the chain is unfinalized, the freeze will have no effect as no honest validator will attempt to build on the frozen chain. However, if the approval checking protocol has failed and the bad parablock is finalized, the freeze serves to put the chain into a governance-only mode.
```

Comment:
Should governance-only mode still roll back to the last unfinalized block?  I suppose we're just in such a mess we'll make humans sort it out and not bother with code for that.  :P

Reply:
I'll address this in a later PR

Reply:
But I don't know if it helps TBH